resources/assets/js/components/ApplicantProfile/Skills/Accordion.tsx

```diff
@@ -0,0 +1,633 @@
+                    data-h2-padding="b(right, .5)"
+                  />
+                  <span data-h2-button-label data-h2-font-color="b(theme-1)">
+                    {intl.formatMessage(messages.skillRemoveLabel)}
+                  </span>
+                </Dialog.Trigger>
+                <Dialog id={`skill-remove-dialog-${skill.id}`}>
+                  <Dialog.Header
+                    data-h2-grid="b(middle, contained, padded, .5)"
+                    data-h2-bg-color="b(theme-1, 1)"
+                  >
+                    <div data-h2-grid-item="b(1of2)" data-h2-align="b(left)">
+                      <Dialog.Title
+                        data-h2-margin="b(all, .5) b(left, 1)"
+                        data-h2-padding="b(left, .5)"
+                        data-h2-font-color="b(white)"
+                        data-h2-font-size="b(h5)"
+                      >
+                        <i
+                          className="fas fa-trash"
+                          data-h2-font-color="b(white)"
+                          data-h2-padding="b(right, .5)"
+                        />
+                        {intl.formatMessage(messages.skillRemoveDialogTitle)}
+                      </Dialog.Title>
+                    </div>
+                    <div data-h2-grid-item="b(1of2)" data-h2-align="b(right)">
+                      <Dialog.ActionBtn
+                        data-h2-button="round, small, solid"
+                        data-h2-margin="b(right, 1)"
+                      >
+                        <i
+                          data-h2-font-size="b(normal)"
+                          data-h2-font-color="b(white)"
+                          className="fas fa-times"
+                          aria-hidden="true"
+                        />
+                      </Dialog.ActionBtn>
+                    </div>
+                  </Dialog.Header>
+                  <Dialog.Content>
+                    <p
+                      data-h2-padding="b(tb, 1), b(rl, 2)"
+                      data-h2-font-size="b(h5)"
+                      data-h2-font-color="b(theme-3)"
+                    >
+                      {intl.formatMessage(messages.skillRemoveDialogHeading)}
+                    </p>
+                    <p data-h2-padding="b(tb, .5), b(rl, 2)">
+                      {intl.formatMessage(messages.skillRemoveDialogParagraph1)}
+                      <br />
+                      <br />
+                      {intl.formatMessage(messages.skillRemoveDialogParagraph2)}
+                    </p>
+                  </Dialog.Content>
+                  <Dialog.Actions
+                    data-h2-padding="b(tb, 1) b(rl, 2)"
+                    data-h2-align="b(center)"
+                    data-h2-grid="b(middle, expanded, flush, 1)"
+                  >
+                    <div data-h2-grid-item="b(1of2)" data-h2-align="b(left)">
+                      <div data-h2-grid-content>
+                        <Dialog.ActionBtn
```

Comment:
Needs an onClick. The SkillAccordion should probably accept a callback prop, something like `deleteSkill: (skillId: number) => Promise<void>`, which can be called when you click this.